app/models/contestant.rb

```diff
@@ -1,3 +1,9 @@
 class Contestant < ApplicationRecord
   validates_presence_of :name, :age, :hometown, :years_of_experience
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
+
+  def project_names
+    self.projects.pluck(:name).join(', ')
```

Comment:
Love the use of this instance method to join the names in a sentence! Excellent encapsulation here